modules/distribution/resources/migration-2.6.0_to_3.0.0/db2.sql

```diff
@@ -68,3 +68,17 @@ CREATE TABLE IF NOT EXISTS AM_NOTIFICATION_SUBSCRIBER (
 ALTER TABLE AM_EXTERNAL_STORES
 ADD LAST_UPDATED_TIME TIMESTAMP DEFAULT CURRENT_TIMESTAMP
 /
+
+ALTER TABLE AM_API
+  ADD API_TYPE VARCHAR(10) NULL DEFAULT NULL
+/
+
+CREATE TABLE AM_API_PRODUCT_MAPPING (
+  API_PRODUCT_MAPPING_ID INTEGER GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+  API_ID INTEGER,
+  URL_MAPPING_ID INTEGER,
+  FOREIGN KEY (API_ID) REFERENCES AM_API(API_ID) ON DELETE CASCADE,
+  FOREIGN KEY (URL_MAPPING_ID) REFERENCES AM_API_URL_MAPPING(URL_MAPPING_ID) ON DELETE CASCADE,
+  PRIMARY KEY(API_PRODUCT_MAPPING_ID)
+)
+  /
```

Comment:
Shall we fix the minor formatting issue here?

Reply:
Resolved